site/content/en/docs/Installation/helm.md

```diff
@@ -80,42 +80,46 @@ The following tables lists the configurable parameters of the Open Match chart a
-| `global.telemetry.grafana.enabled`                      | Turn on/off Open Match Grafana exporter. Also install Grafana if `open-match-telemetry.enabled` is set to true                                                          | `false`         |
-| `global.telemetry.reportingPeriod`                       | Overrides the reporting periods of Open Match telemetry exporters                                              | `1m`                 |
-| `open-match-telemetry.grafana`       | Inherits the values from [Grafana helm chart](https://github.com/helm/charts/tree/master/stable/grafana)                                               |                     |
-| `open-match-telemetry.jaeger`    | Inherits the values from [Jaeger helm chart](https://github.com/helm/charts/tree/master/incubator/jaeger)                                                   |                     |
-| `open-match-telemetry.prometheus`      | Inherits the values from [Prometheus helm chart](https://github.com/helm/charts/tree/master/stable/prometheus)                                  |                   |
-| `redis` | Inherits the values from [Redis helm chart](https://github.com/helm/charts/tree/master/stable/redis)                                    |                |
-
+| `query.portType` | Defines Kubernetes [ServiceTypes](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) for the QueryService | `ClusterIP` |
+| `query.replicas` | Defines the number of pod replicas for QueryService's Kubernetes deployment | `3` |
+| `frontend.portType` | Defines Kubernetes [ServiceTypes](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) for the FrontendService | `ClusterIP` |
+| `frontend.replicas` | Defines the number of pod replicas for FrontendService's Kubernetes deployment | `3` |
+| `backend.portType` | Defines Kubernetes [ServiceTypes](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) for the BackendService | `ClusterIP` |
+| `backend.replicas` | Defines the number of pod replicas for BackendService's Kubernetes deployment | `3` |
+| `image.pullPolicy` | Global `imagePullPolicy` for all Open Match service deployments | `Always` |
+| `image.tag` | Global Docker image tag for all Open Match service deployments | `{{< param release_version >}}` |
+| `open-match-core.enabled` | Turn on/off the installation of Open Match core services | `true` |
+| `open-match-core.registrationInterval` | Duration of registration window for evaluation/synchronization cycle. | `250ms` |
+| `open-match-core.proposalCollectionInterval` | Length of time after match function as started before it will be canceled. | `20s` |
```

Comment:
Length of time after match function **has** started before it will be canceled

Reply:
missed that typo, thanks!